draft-yasskin-http-origin-signed-responses.md

```diff
@@ -286,18 +288,15 @@ extended diagnostic notation from {{?I-D.ietf-cbor-cddl}} appendix G:
 
 ~~~cbor-diag
 [
-  "request",
-  [
-    ':method', 'GET',
-    ':url', 'https://example.com/'
-  ],
-  "response",
-  [
-    ':status', '200',
-    'content-type', 'text/html'
-  ],
-  "payload",
-  '<!doctype html>\n<html>...'
+  {
+    ':url': 'https://example.com/'
+    ':method': 'GET',
```

Comment:
I assume the keys need to be sorted?
https://wicg.github.io/webpackage/draft-yasskin-http-origin-signed-responses.html#canonical-cbor

Reply:
sorry maybe this is already sorted in "the bytewise lexicographic order"

Reply:
Exactly: because the encoding of a string starts with its length, shorter strings sort before longer ones. This isn't obvious from the definition, but I do have the example of "z" sorting before "aa".

There's no actual need for the diagnostic form to appear in sorted order anyway, since canonicalization happens when encoding a semantic CBOR item to bytes, but I did sort it just to demonstrate the order.